docker-compose.yml

```diff
@@ -1,29 +1,32 @@
-version: '3.2'
+version: "3.2"
 services:
   postgres:
     image: "postgres:12"
     ports:
       - "5432:5432"
     volumes:
-    - type: bind
-      source: ./volumes/postgres
-      target: /var/lib/postgresql/data
+      - type: bind
+        source: ./volumes/postgres
+        target: /var/lib/postgresql/data
     environment:
-        - POSTGRES_HOST_AUTH_METHOD=trust
+      - POSTGRES_HOST_AUTH_METHOD=trust
 
   rskj:
-    build:
-      context: ./docker/rskj
+    image: rsksmart/rskj:HOP-4.1.1
     ports:
       - "4444:4444"
       - "4445:4445"
     volumes:
       - type: bind
         source: ./volumes/rskj
         target: /etc/rsknode
-  
+      - ./docker/rskj/logback.xml:/etc/rsk/logback.xml
+      - ./docker/rskj/node.conf:/etc/rsk/node.conf
+
   dev-liquidity-token-watcher:
-    image: "matterlabs/dev-liquidity-token-watcher:latest"
+    # image: "matterlabs/dev-liquidity-token-watcher:latest"
```

Comment:
I'd remove it if not necessary, or we add a more meaningful comment, like `TODO: Once we publish the image, we could use it directly, instead of building it each time.